### ISSUE TITLE
Filtering on types

### ISSUE THREAD

**Issue by gummiboll** [linode/python-linode-api#47]
Hello!

Filtering with `get_types()` on linode types doesn't work according to the documentation (https://developers.linode.com/v4/libraries/python/linode-client):

```
>>> tps = c.linode.get_types(linode.Types.label == "Linode 1024")[0]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'Types'
```

**Comment by Dorthu** [comment]
I'll take a look, thanks for reporting

**Comment by Dorthu** [comment]
Right now, the "type" object is misnamed "Service" - submitting a patch that renames it to "Type".  Once this is release, this will work:

`tps = lc.linode.get_types(linode.Type.label == "Linode 1024").first()`

For convenience, you could accomplish the same thing by just using the linode type's slug (and without waiting for the release):

`tps = "g5-nanode-1"`

The slug should be accepted everywhere in place of the Type object.

**Comment by Dorthu** [comment]
This fix has been released in v4.1.3b1
